src/dynarec/rv64/dynarec_rv64_00_3.c

```diff
@@ -899,25 +899,23 @@ uintptr_t dynarec64_00_3(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
                         AND(xRAX, x2, xMASK);
                         ZEROUP(xRDX);
                     } else {
-                        if(ninst 
-                           && dyn->insts[ninst-1].x64.addr 
-                           && *(uint8_t*)(dyn->insts[ninst-1].x64.addr)==0x31 
+                        if(ninst
+                           && dyn->insts[ninst-1].x64.addr
+                           && *(uint8_t*)(dyn->insts[ninst-1].x64.addr)==0x31
                            && *(uint8_t*)(dyn->insts[ninst-1].x64.addr+1)==0xD2) {
                             SET_DFNONE();
                             GETED(0);
-                            DIVU(x2, xRAX, ed);
                             REMU(xRDX, xRAX, ed);
```

Comment:
but what if ed is RDX?

Reply:
Hmmm, indeed, I never thought about this, sorry.

Reply:
> but what if ed is RDX (this is why the code was like that)

We can use `if (ed == xRDX)` to save 1 instruction when ed is not xRDX, but I don't think it's worth it, I'll just revert this change.